Nutraceuticals section added (#122)
greenelab/covid19-review: content/20.treatments.md

#### Broad-Spectrum Pharmaceuticals

##### Nutraceuticals
Considering the current pandemic, scientists and the medical community are scrambling to repurpose or discover novel host-directed therapies for which nutraceuticals hold some promise.
Nutraceuticals are classified as supplements with health benefits beyond their basic nutritional value designed for the prophylaxis and treatment of disease [@doi:10.1080/10408390902841529; @doi:10.1038/nrcardio.2016.103].

> Nutraceuticals purported to boost the immune response, reduce  immunopathology, exhibit antiviral activities or prevent acute respiratory distress syndrome (ARDS) are being considered for their potential therapeutic value [@doi:ggkd3b].

Reviewer: ```suggestion
Nutraceuticals purported to boost the immune response, reduce  immunopathology, exhibit antiviral activities or prevent acute respiratory distress syndrome (ARDS) are being considered for their potential therapeutic value [@doi:10/ggkd3b].
```

A host of potential candidates have been highlighted in the literature that target various aspects of the COVID-19 viral pathology, while others are thought to prime the host immune system.

> These include vitamins and minerals along with extracts and omega-3 polyunsaturated fatty acids (n-3 PUFA) [@doi:10.1002/jmv.25707].

Reviewer: ```suggestion
These candidates include vitamins and minerals along with extracts and omega-3 polyunsaturated fatty acids (n-3 PUFA) [@doi:10.1002/jmv.25707].
```
It's not totally clear to me if that is what you meant, so please fix if I'm misunderstanding!

> These include vitamins and minerals along with extracts and omega-3 polyunsaturated fatty acids (n-3 PUFA) [@https://doi:10.1002/jmv.25707].

Author: Thanks @cgreene apologies for that

Considerable evidence in vitro and in vivo suggests that nutraceuticals containing phycocyanobilin, N-acetylcysteine, glucosamine, selenium or phase 2 inductive nutraceuticals (e.g. ferulic acid, lipoic acid, or sulforaphane) can prevent or modulate RNA virus infections via amplification of the signaling activity of mitochondrial antiviral-signaling protein (MAVS) and activation of toll-like receptor 7 (TLR7) [@doi:10.1016/j.pcad.2020.02.007].

Another potential nutraceutical that has exhibited beneficial effects against various viral infections is n-3 PUFA [@doi:10.1002/jmv.25707], such as eicosapentaenoic acid (EPA) and docosahexaenoic acid (DHA).

> N-3 PUFA nutraceuticals can mediate inflammation and they have the capacity to modulate the adaptive immune response [@doi:10.1111/j.1365-2125.2012.04374.x; @doi:10.3390/molecules22111964; @doi:10.1038/nrcardio.2016.103].

Reviewer: You are definitely the expert here, especially as you authored one of the citations, but the first reference made it sound like the research showing this has largely been in animal models? I just wanted to check if that's true.

Author: Thanks for checking. The majority of the mechanistic research has largely been in animal models. There is mixed research in this field because the potential beneficial effects of n-3 fatty acids is very pathogen specific, in some cases it is suggested they do nothing, in others they might do more harm than good. In terms of coronavirus there is promise because of the cardiovascular pathology involved. But on that point I might see if I can find a few human studies or a meta-analysis for here

One potential mechanism by which n-3 PUFA could exert beneficial effects against viral infections is by acting as precursor molecules for the biosynthesis of endogenous specialised proresolving mediators (SPM) like protectins and resolvins that actively resolve inflammation and infection [@https://dx.doi.org/10.1016/j.immuni.2014.02.009]. 
SPM have exhibited beneficial effects against a variety of lung infections including RNA viruses [@https://doi.org/10.1038/nri.2015.4].
Indeed, protectin D1 has been shown to increase survival from H1N1 viral infection in mice by affecting the viral replication machinery [@https://doi.org/10.1016/j.cell.2013.02.027].
Overall, there is currently limited evidence that these nutraceuticals may affect COVID-19 infection as there is a significant lack of human studies.  

In contrast, there is evidence that nutrient supplements may exhibit some benefit against RNA viral infections. 
Zinc is a trace metal obtained from dietary sources or supplementation that is important for the maintenance of immune cells involved in adaptive and innate immunity [@https://doi.org/10.1016/j.abb.2016.03.022].

> Indeed, adequate zinc intake has been associated with reduced incidence of infection [@https://doi.org/10.1093/ajcn/85.3.837] and antiviral immunity [@https://doi.org/10.1093/advances/nmz013].

Reviewer 2: ```suggestion
Indeed, adequate zinc intake has been associated with reduced incidence of infection [@doi:10.1093/ajcn/85.3.837] and antiviral immunity [@doi:10.1093/advances/nmz013].
```

> Clinical trial data supports the utility of zinc to diminish the duration and severity of symptoms associated with common colds when it is provided within 24 hours of the onset of symptoms [@https://doi.org/10.1331/1544-3191.44.5.594.Hulisz; [@https://doi.org/10.2174/1874306401105010051].

Reviewer 2: ```suggestion
Clinical trial data supports the utility of zinc to diminish the duration and severity of symptoms associated with common colds when it is provided within 24 hours of the onset of symptoms [@doi:10.1331/1544-3191.44.5.594.Hulisz; @doi:10.2174/1874306401105010051].
```

> In coronaviruses specifically, in vitro evidence demonstrates that the combination of zinc (Zn2+) and zinc ionophores (pyrithione) can interrupt the replication mechanisms of SARS-CoV-GFP and a variety of other RNA viruses [@doi:10.1371/journal.ppat.1001176; @doi:10.1016/j.antiviral.2014.12.015]. 

Reviewer: From looking into it, it seems like SARS-CoV-GFP is just a fluorescently tagged version of SARS-CoV, but I initially assumed it was a different strain -- it might be worth clarifying in the text
I'm curious whether there's an association between nutrition status and intracellular zinc levels? I assume we don't know whether zinc supplementation or diet modulates zinc concentration at the cellular level?

Author: Good question, there is limited evidence from what I have read. From what I have read previously it is specific to the target organ. I will look into this

Author: So yes, nutritional status intracellular zinc levels are linked does - Examples of zinc supplementation increasing plasma zinc levels and other tissues in uremia patients (https://academic.oup.com/ajcn/article-abstract/36/6/1177/4693588). 

Indeed, it is well covered in infection, for example: https://academic.oup.com/ajcn/article/85/3/837/4633003

I will add this to the section as it is a very interesting point to make.

> In coronaviruses specifically, in vitro evidence demonstrates that the combination of zinc (Zn2+) and zinc ionophores (pyrithione) can interrupt the replication mechanisms of SARS-CoV-GFP and a variety of other RNA viruses [@doi:10.1371/journal.ppat.1001176; @doi:10.1016/j.antiviral.2014.12.015]. 

Author: In coronaviruses specifically, in vitro evidence demonstrates that the combination of zinc (Zn2+) and zinc ionophores (pyrithione) can interrupt the replication mechanisms of SARS-CoV-GFP (a florescently tagged SARS-CoV) and a variety of other RNA viruses [@doi:10.1371/journal.ppat.1001176; @doi:10.1016/j.antiviral.2014.12.015].

> In coronaviruses specifically, in vitro evidence demonstrates that the combination of zinc (Zn2+) and zinc ionophores (pyrithione) can interrupt the replication mechanisms of SARS-CoV-GFP and a variety of other RNA viruses [@doi:10.1371/journal.ppat.1001176; @doi:10.1016/j.antiviral.2014.12.015]. 

Author: ```suggestion
In coronaviruses specifically, in vitro evidence demonstrates that the combination of zinc (Zn2+) and zinc ionophores (pyrithione) can interrupt the replication mechanisms of SARS-CoV-GFP and a variety of other RNA viruses [@doi:10.1371/journal.ppat.1001176; @doi:10.1016/j.antiviral.2014.12.015]. 
```

> In coronaviruses specifically, in vitro evidence demonstrates that the combination of zinc (Zn2+) and zinc ionophores (pyrithione) can interrupt the replication mechanisms of SARS-CoV-GFP and a variety of other RNA viruses [@https://doi.org/10.1371/journal.ppat.1001176; @http://dx.doi.org/10.1016/j.antiviral.2014.12.015]. 

Reviewer 2: ```suggestion
In coronaviruses specifically, in vitro evidence demonstrates that the combination of zinc (Zn2+) and zinc ionophores (pyrithione) can interrupt the replication mechanisms of SARS-CoV-GFP and a variety of other RNA viruses [@doi:10.1371/journal.ppat.1001176; @doi:10.1016/j.antiviral.2014.12.015]. 
```

While overall there is encouraging data for zinc supplementation against the common cold, there is currently limited evidence to suggest zinc supplementation has any beneficial effects against the current novel COVID-19, thus further research is imperative.

Vitamins B, C, D, and E have also been suggested as potential nutrient supplement interventions for COVID-19 [@doi:10.1002/jmv.25707; @doi:10.20944/preprints202003.0347.v1].

> Despite that it is well documented that regular vitamin C supplementation may reduce the duration of the common cold, vitamin C (> 200 mg) fails to reduce the incidence of  [@doi:10.1002/14651858.CD000980.pub4.].

Author: Thanks @rando2 all help appreciated

Despite these findings, the CITRUS ALI study failed to show a benefit of a 96-hour infusion of vitamin C to treat ARDS, which is a severe complication of COVID-19 infection [@doi:10.1001/jama.2019.11825].
There is anecdotal evidence from China and the United States that vitamin C is currently being used for the treatment of COVID-19, but further controlled research is required.    

In terms of other dietary supplements, vitamin D can modulate the adaptive and innate immune system.
Vitamin D deficiency is associated with an increased susceptibility to infection [@doi:10.231/JIM.0b013e31821b8755].
A recent preprint postulated that an individual’s vitamin D status may significantly affect one’s risk of developing COVID-19. 
This hypothesis was derived from the fact that the current pandemic occurred in winter in Wuhan China when 25-hydroxyvitamin D concentrations are at their lowest due to lack of sunlight, whereas in the Southern Hemisphere, where it was nearing the end of the summer, the number of cases was low at that time coinciding with higher 25-hydroxyvitamin D concentrations  [@doi:10.20944/preprints202003.0235.v2].

> However, this has yet to be determined as it is not yet even known whether COVID-19 is seasonal. 

Reviewer: I only skimmed this preprint, but I was wondering if it might be worth mentioning that there are many factors that correlate with season (and old age), one of which is population-scale vitamin D levels, so the vitamin D --> infection risk causal relationship that study is putting forward is one explanation among many possibilities for why the correlation might be observed. It also sounds like many of the background studies showed that having a deficiency vs not having a deficiency may be the important factor? I have to admit, I'm a bit skeptical of this preprint just based on their conflicts of interest, so apologies if I'm missing something!

Author: Good suggestions, I had thought to expand this further. The only reason I didn't is I thought I had gone too far! But I think you are right, I will make an edit on that, feel free to edit

Certainly, deficiency in vitamin D has been associated with a risk of ARDS and respiratory infection [@doi:10.1016/j.jsbmb.2012.11.017]. 

> Therefore, the utility of supplementation and maintaining a healthy diet for optimum vitamin D status warrants further investigation. This is particularly important considering ‘stay in place’ guidance has been implemented in many densely populated cities around the world. 

Reviewer: ```suggestion
Therefore, the utility of supplementation and maintaining a healthy diet for optimum vitamin D status warrants further investigation. 
This is particularly important considering ‘stay in place’ guidance has been implemented in many densely populated cities around the world. 
```

> This measure will limit people’s exposure to sunlight and thus reduce endogenous synthesis of vitamin D weakening the immune system and potentially increasing the risk of COVID-19 infection. 

Reviewer: ```suggestion
This measure is likely to limit people’s exposure to sunlight and thus reduce endogenous synthesis of vitamin D, potentially weakening the immune system and increasing the risk of COVID-19 infection. 
```

Despite all the potential benefits of nutraceutical and dietary supplement interventions presented, currently there is a paucity of clinical evidence to support their use for the prevention or mitigation of COVID-19 infections. 

> However, optimal nutritional status will undoubtedly prime an individual’s immune system to protect against the effects of acute respiratory viral infections by supporting normal maintenance of the immune system (preprint without DOI: Calder, P.C.; Carr, A.C.; Gombart, A.F.; Eggersdorfer, M. Optimal Nutritional Status for a Well-Functioning Immune System is an Important Factor to Protect Against Viral Infections. Preprints 2020, 2020030199). 

Reviewer: ```suggestion
However, optimal nutritional status will undoubtedly prime an individual’s immune system to protect against the effects of acute respiratory viral infections by supporting normal maintenance of the immune system (preprint without DOI: Calder, P.C.; Carr, A.C.; Gombart, A.F.; Eggersdorfer, M. Optimal Nutritional Status for a Well-Functioning Immune System is an Important Factor to Protect Against Viral Infections. Preprints 2020, 2020030199). 
```
```suggestion
However, optimal nutritional status will undoubtedly prime an individual’s immune system to protect against the effects of acute respiratory viral infections by supporting normal maintenance of the immune system [@url:https://www.preprints.org/manuscript/202003.0199/v1].
```
Let's try a URL and see how it looks -- we can manually edit if needed

Zinc supplements can be administered orally as a tablet or as a lozenge and they are available in many forms such zinc picolinate, zinc acetate, and zinc citrate.
Zinc is also available from dietary sources including meat, seafood, nuts, seeds, legumes, and dairy.

###### Potential Mechanisms

The role of zinc in immune function has been extensively reviewed [@doi:10.1016/j.abb.2016.03.022].
Zinc is an important signaling molecule and zinc levels can alter host defense systems. 
In inflammatory situations such as an infection, zinc can regulate leukocyte immune responses and it can activate the nuclear factor kappa-light-chain-enhancer of activated B cells (NF-kB), thus altering cytokine production [@doi:10.4049/jimmunol.179.6.4180; @doi:10.1067/mlc.2001.118108]. 
In particular, zinc supplementation can increase natural killer cell levels, which are important cells for host defense against viral infections [@doi:10.1126/science.1198687; @doi:10.1016/j.abb.2016.03.022].

###### Current Evidence

Adequate zinc intake has been associated with reduced incidence of infection [@doi:10.1093/ajcn/85.3.837] and antiviral immunity [@doi:10.1093/advances/nmz013].

> Conversely, zinc deficiency is associated with increased susceptibility to infection in the elderly, which can be prevented when zinc supplements are administered [@doi:10.1093/ajcn/85.3.837].

Reviewer: ```suggestion
Similarly, a randomized, double-blind, placebo-controlled trial that administered zinc supplementation to elderly subjects over the course of a year found zinc deficiency to be associated with increased susceptibility to infection and that zinc deficiency could be prevented through supplementation [@doi:10.1093/ajcn/85.3.837].
```
Apologies, I have a few more questions!
1. Is "conversely" this right word? The framing is swapped between the two sentences, but the ideas don't seem at odds -- I'm reading this as both the general population and the elderly in particular need to avoid zinc deficiency. 
2. I checked out that paper because I was curious. It seems pretty well designed to my eye? Again, I've been a genomics person for the last ~8 years, it's been a while since I thought about anything double-blind or placebo-controlled! But I fleshed out the description a little, don't feel like you have to accept!
3. As someone who knows almost nothing about neutraceuticals, I know being vitamin deficient is bad for your health (e.g., scurvy and vitamin C), but what I am unsure of is what kinds of supplementation can help -- namely, do modern vitamin C supplements do anything or should we all just be trying to eat limes? If this study showed that non-diet zinc supplementation actually helps prevent zinc deficiency, and also that zinc deficiency can affect disease risk, I personally think that's an interesting detail!
3. Looking at this paper, it actually seems quite closely tied in with some of the other sections that have touched on cytokines (for pharmaceuticals). We might want to keep an eye on flow between these two sections since they seem closely related!

Clinical trial data supports the utility of zinc to diminish the duration and severity of symptoms associated with common colds when it is provided within 24 hours of the onset of symptoms [@doi:10.1331/1544-3191.44.5.594.Hulisz; @doi:10.2174/1874306401105010051].
In coronaviruses specifically, in vitro evidence demonstrates that the combination of zinc (Zn2+) and zinc ionophores (pyrithione) can interrupt the replication mechanisms of SARS-CoV-GFP (a flourescently tagged SARS-CoV) and a variety of other RNA viruses [@doi:10.1371/journal.ppat.1001176; @doi:10.1016/j.antiviral.2014.12.015].

###### Summary

While overall there is encouraging data for zinc supplementation against the common cold and viral infections, there is currently limited evidence to suggest zinc supplementation has any beneficial effects against the current novel COVID-19, thus further research is imperative.

> However, it would be advisable to maintain a healthy diet to ensure an adequate zinc status to prevent the likelyhood of an infection. 

Reviewer: ```suggestion
However, it would be advisable to maintain a healthy diet to ensure an adequate zinc status to prevent the likelihood of an infection. 
```

##### Vitamin C

Vitamins B, C, D, and E have also been suggested as potential nutrient supplement interventions for COVID-19 [@doi:10.1002/jmv.25707; @doi:10.20944/preprints202003.0347.v1].

> In particular vitamin C has been considered as a potential prophylactic and therapeutic agent of COVID-19. 

Reviewer: ```suggestion
In particular vitamin C has been proposed as a potential prophylactic and therapeutic agent against COVID-19. 
```

> Vitamin C can also be obtained via dietary sources such as fruit and vegetable or via supplementation.  

Reviewer: ```suggestion
Vitamin C can be obtained via dietary sources such as fruit and vegetable or via supplementation.  
```

###### Potential Mechanisms
Vitamin C plays a signifiant role in promoting immune function due to its effects on various immune cells. 
Vitamin C affects inflammation by modulating cytokine production, decreasing histamine levels, enhancing the differentiation and proliferation of T- and B-lymphocytes, increasing antibody levels, and protecting against the negative effects of reactive oxygen species amongst other effects [@doi:10.1155/2014/426740; @doi:10.1007/s00210-013-0880-1; @doi:10.3390/nu9111211]. 

> During viral infections vitamin C is utilised as evinced by lower concentrations in leukocytes and lower concentrations of urinary vitamin C. Post infection these levels return to baseline ranges [@doi:10.1177/003693307301800102; @doi:10.1111/j.1749-6632.1975.tb29312.x; @doi:10.1016/0006-2944(79)90058-9; @doi:10.3390/nu9040339; @doi:10.1079/bjn19920004]. 

Reviewer: ```suggestion
During viral infections vitamin C is utilised as evinced by lower concentrations in leukocytes and lower concentrations of urinary vitamin C. 
Post-infection, these levels return to baseline ranges [@doi:10.1177/003693307301800102; @doi:10.1111/j.1749-6632.1975.tb29312.x; @doi:10.1016/0006-2944(79)90058-9; @doi:10.3390/nu9040339; @doi:10.1079/bjn19920004]. 
```

###### Current Evidence

> Despite the fact that it is well documented that regular vitamin C supplementation can reduce the duration of the common cold, vitamin C (> 200 mg) fails to reduce its incidence [@doi:10.1002/14651858.CD000980.pub4].

Reviewer: ```suggestion
A recent meta-analysis found consistent support for regular vitamin C supplementation reducing the duration of the common cold, but that supplementation with vitamin C (> 200 mg) failed to reduce the incidence of colds [@doi:10.1002/14651858.CD000980.pub4].
```

> Vitamin C has been shown to reduce the susceptibility of patients to lower respiratory tract infections such as pneumonia [@doi:10.1097/00006454-199709000-00003].

Reviewer: ```suggestion
Individual studies have found Vitamin C to reduce the susceptibility of patients to lower respiratory tract infections such as pneumonia [@doi:10.1097/00006454-199709000-00003].
```
My outsider's understanding is that plenty of studies have reported effects of Vitamin C but many don't hold up at the meta-analysis level -- if that's correct, I think it would be good to distinguish meta-analyses from studies, as you do in the lines below

While vitamin D is relatively inexpensive and safe to consume, caution is warranted when interpreting this review as it has yet to be determined whether vitamin D levels effect COVID-19 specifically. 
Likewise, it is assumed that COVID-19 may be seasonal, but there are multiple other factors at play that can affect vitamin D levels that need to be considered.
These factors include an individual’s nutritional status, their age, their occupation, skin pigmentation, potential comorbidities, and the variation of exposure to sunlight due to latitude amongst others [@doi:10.3390/nu7075111].   

###### Summary
  
Supplementation of vitamin D and maintaining a healthy diet for optimum vitamin D status warrants further investigation. 
This is particularly important considering ‘stay in place’ guidance has been implemented in many densely populated cities around the world.
This measure is likely to limit people’s exposure to sunlight and thus reduce endogenous synthesis of vitamin D, potentially weakening the immune system and increasing the risk of COVID-19 infection. 

##### Nutraceutical Conclusions

> Despite all the potential benefits of nutraceutical and dietary supplement interventions presented, currently there is a paucity of clinical evidence to support their use for the prevention or mitigation of COVID-19 infections. 

Reviewer: ```suggestion

Despite all the potential benefits of nutraceutical and dietary supplement interventions presented, currently there is a paucity of clinical evidence to support their use for the prevention or mitigation of COVID-19 infections. 
```
